modules/react/text-area/lib/TextArea.tsx

```diff
@@ -37,71 +38,59 @@ export interface TextAreaProps extends Themeable, GrowthBehavior {
    * The resize constraints of the TextArea.
    * @default TextArea.ResizeDirection.Both
    */
-  resize?: TextAreaResizeDirection;
+  resize?: ValueOf<typeof TextAreaResizeDirection>;
   /**
    * The value of the TextArea.
    */
   value?: any;
 }
 
-export enum TextAreaResizeDirection {
-  None = 'none',
-  Both = 'both',
-  Horizontal = 'horizontal',
-  Vertical = 'vertical',
-}
+export const TextAreaResizeDirection = {
+  None: 'none' as 'none',
+  Both: 'both' as 'both',
+  Horizontal: 'horizontal' as 'horizontal',
+  Vertical: 'vertical' as 'vertical',
+};
```

Comment:
```suggestion
export const TextAreaResizeDirection = {
  None: 'none',
  Both: 'both',
  Horizontal: 'horizontal',
  Vertical: 'vertical',
} as const;
```

Does it do the same?

Reply:
it's like, so simple, it's beautiful